release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -154,6 +154,12 @@ If you need the virtual IP (VIP) addresses to run on the control plane nodes in
 [id="ocp-4-13-post-installation"]
 === Post-installation configuration
 
+[id="ocp-4-13-multi-arch-compute-machines-ga"]
+==== {product-title} clusters with multi-architecture compute machines 
+
+{product-title} {product-version} clusters with multi-architecture compute machines is now generally available. You can now create a cluster with compute nodes of different architectures on AWS and Azure installer provisioned infrastructures. Bare metal user provisioned installations are in Technology Preview. For more information on creating a cluster with multi-architecture compute machines, see xref:../post_installation_configuration/multi-architecture-configuration.adoc#multi-architecture-configuration[Configuring multi-architecture compute machines on an OpenShift Container Platform cluster].
```

Comment:
lgtm ....

```
You can now create a cluster with compute nodes of different architectures on AWS and Azure installer provisioned infrastructures
````
yes we do provision with IPI but the mixed arch compute nodes are added as a day2 operation right? is there a succinct way to mention this?